product_docs/docs/efm/4/04_configuring_efm/05_using_vip_addresses.mdx

```diff
@@ -107,9 +111,10 @@ When instructed to ping the VIP from a node, use the command defined by the `pin
    rtt min/avg/max/mdev = 0.023/0.025/0.029/0.006 ms
    ```
 
-   No packet loss occurs. 
+   You will see 0% packet loss indicating the IP now reaches the primary node.
```

Comment:
I left a note here that I no longer see. This change isn't correct: there is no primary node in the testing scenario.

Reply:
That conflicts with step #2 above then, which has you running the script from the primary node!

Reply:
As discussed in slack, we will not handle any changes related to testing in this PR. Only the notes about not dropping the VIP at shutdown are relevant here.